src/dmtest/vdo/utils.py

```diff
@@ -28,3 +50,25 @@ def fsync(dev):
     """Sync the specified device or file."""
     with open(dev, 'w') as thing:
         os.fsync(thing.fileno())
+
+def run_fio(dev, size, offset, verify = False, stats = True):
+    fio_config = fio_config_template.format(size=size,
+                                            offset=offset,
+                                            compress=74,
```

Comment:
Hard-coding compressibility at 74% made sense for the compression test, but I think not for a general support routine, as it holds up I/Os in the packer and makes some of the stats harder to predict if you're not careful.

Reply:
Will be fixed in next push.

Reply:
I've made compression configurable, and all tests pass.  Interestingly, this made the new test start failing.  I'm guessing because of the way compressed blocks are generated?

So as a result, I've added `randseed=1` (do we want this to be more unique run to run?) and `refill_buffers=1` to get repeatable datasets from run to run.

Reply:
@raeburn , your comment got pulled out of being in-line due to email response... 

> On Tue, Jul 30, 2024 at 9:29 PM Andy Walsh ***@***.***> wrote: ***@***.**** commented on this pull request. ------------------------------ I've made compression configurable, and all tests pass. Interestingly, this made the new test start failing. I'm guessing because of the way compressed blocks are generated? So as a result, I've added randseed=1 (do we want this to be more unique run to run?) and refill_buffers=1 to get repeatable datasets from run to run.
> We could add a randomized seed for each run (and log it, and have the ability to reproduce a given run?), but I don't think setting a fixed value is any different from just setting randrepeat=1. (Or the man page needs some clarification...)
> […](#)
> Message ID: ***@***.***>

I just tested, I can drop the `randseed=1`.  So long as I retain `refill_buffers=1`, then the new test passes.  Is that what you would like for me to do here?

Reply:
Yes, that looks like the right thing

Reply:
ok that has now been addressed in the most recent push.